frontend/src/component/FeedItem/FeedItemFooter.jsx

```diff
@@ -62,7 +62,7 @@ export class FeedItemFooter extends React.Component {
           {props.isPublication && props.showTimeline && (
             <IconButton
               text={props.discussionToggleButtonLabel}
-              textMobile={props.discussionToggleButtonLabelMobile}
+              textMobile={props.discussionToggleButtonLabelMobile.toString()}
```

Comment:
```suggestion
              textMobile={props.discussionToggleButtonLabelMobile}
```
According to your changement on `FeedItemWithPreview`, discussionToggleButtonLabelMobile` is already a string.

Reply:
done